tap_hubspot_beta/streams.py

```diff
@@ -1050,6 +1050,74 @@ def get_url_params(self, context, next_page_token):
             params["properties"] = "id,createdAt,updatedAt,archived,archivedAt"
         return params
 
+class ArchivedProductsStream(hubspotV3Stream):
```

Comment:
The parent class doesn't match the one from the other archived streams, it should be `ArchivedStream` I also think if you use this parent stream, some of your code is no longer needed so you can clean it up a bit. Refer to the class above as an example